packages/components/src/base-control/README.md

```diff
@@ -8,17 +8,19 @@ BaseControl component is used to generate labels and help text for components ha
 
 Render a BaseControl for a textarea input:
 ```jsx
-	<BaseControl
```

Comment:
Can we also include import statement to make it possible to extract code as is and run it without any modifications?

Reply:
Yeah, that would be great but I'm afraid I have a limitation here. 

I'm using [`react-live`](https://github.com/FormidableLabs/react-live) for compiling and running these snippets of code in real time, so we avoid to change our build process.

That forces us to import the dependencies outside the snippets in order to define the scope needed by `react-live` to run the extracted code.

Reply:
Anyway, I think we should maintain them in the examples in order to provide documentation as complete as possible. We can always ignore them when running the snippets (you can check how I'm handling that here: https://github.com/Automattic/wp-calypso/pull/26367/files#diff-d0c3bde7d6504cb5de69c0488b877342R36)